src/python/pants/backend/docker/lint/hadolint/subsystem.py

```diff
@@ -14,20 +14,20 @@ class Hadolint(TemplatedExternalTool):
     name = "Hadolint"
     help = "A linter for Dockerfiles."
 
-    default_version = "v2.8.0"
-    # TODO: https://github.com/hadolint/hadolint/issues/411 tracks building and releasing
-    #  hadolint for Linux ARM64.
+    default_version = "v2.10.0"
     default_known_versions = [
-        "v2.8.0|macos_x86_64|27985f257a216ecab06a16e643e8cb0123e7145b5d526cfcb4ce7a31fe99f357|2428944",
-        "v2.8.0|macos_arm64 |27985f257a216ecab06a16e643e8cb0123e7145b5d526cfcb4ce7a31fe99f357|2428944",  # same as mac x86
-        "v2.8.0|linux_x86_64|9dfc155139a1e1e9b3b28f3de9907736b9dfe7cead1c3a0ae7ff0158f3191674|5895708",
+        "v2.10.0|macos_x86_64|59f0523069a857ae918b8ac0774230013f7bcc00c1ea28119c2311353120867a|2514960",
+        "v2.10.0|macos_arm64 |59f0523069a857ae918b8ac0774230013f7bcc00c1ea28119c2311353120867a|2514960",  # same as mac x86
+        "v2.10.0|linux_x86_64|8ee6ff537341681f9e91bae2d5da451b15c575691e33980893732d866d3cefc4|2301804",
+        "v2.10.0|linux_arm64 |b53d5ab10707a585c9e72375d51b7357522300b5329cfa3f91e482687176e128|27954520",
     ]
     default_url_template = (
         "https://github.com/hadolint/hadolint/releases/download/{version}/hadolint-{platform}"
     )
     default_url_platform_mapping = {
         "macos_arm64": "Darwin-x86_64",
         "macos_x86_64": "Darwin-x86_64",
+        "linux_arm64": "linux.aarch64",
```

Comment:
I think this right? Per https://github.com/hadolint/hadolint/releases/tag/v2.10.0

```suggestion
        "linux_arm64": "Linux-arm64",
```

Reply:
Ah, I misunderstood what this was for and had copied what pants did for shellshock. I've pushed up a fix for this. Thanks!